dpgen/data/arginfo.py

```diff
@@ -35,6 +35,135 @@ def init_reaction_mdata_arginfo() -> Argument:
+    doc_pert_numb = "Number of perturbations for each POSCAR."
+    doc_pert_box = "Percentage of Perturbation for cells."
+    doc_pert_atom = "Perturbation of each atoms (Angstrom)."
+    doc_md_nstep = "Steps of AIMD in stage 3. If it's not equal to settings via `NSW` in `md_incar`, DP-GEN will follow `NSW`."
+    doc_coll_ndata = "Maximal number of collected data."
+    doc_type_map = "The indices of elements in deepmd formats will be set in this order."
+
+    return Argument("init_bulk_jdata", dict, [
+        Argument("stages", list, optional=False, doc=doc_stages),
+        Argument("elements", list, optional=False, doc=doc_elements),
+        Argument("potcars", list, optional=True, doc=doc_potcars),
+        Argument("cell_type", str, optional=True, doc=doc_cell_type),
+        Argument("super_cell", list, optional=False, doc=doc_super_cell),
+        Argument("from_poscar", bool, optional=True, default=False, doc=doc_from_poscar),
+        Argument("from_poscar_path", str, optional=True, doc=doc_from_poscar_path),
+        Argument("relax_incar", str, optional=True, doc=doc_relax_incar),
+        Argument("md_incar", str, optional=True, doc=doc_md_incar),
+        Argument("scale", list, optional=False, doc=doc_scale),
+        Argument("skip_relax", bool, optional=False, doc=doc_skip_relax),
+        Argument("pert_numb", int, optional=False, doc=doc_pert_numb),
+        Argument("pert_box", float, optional=False, doc=doc_pert_box),
+        Argument("pert_atom", float, optional=False, doc=doc_pert_atom),
+        Argument("md_nstep", int, optional=False, doc=doc_md_nstep),
+        Argument("coll_ndata", int, optional=False, doc=doc_coll_ndata),
+        Argument("type_map", list, optional=True, doc=doc_type_map),
+    ], sub_variants=init_bulk_variant_type_args(),
+    doc=doc_init_bulk) 
+
+def init_surf_jdata_arginfo() -> Argument:
+    """Generate arginfo for dpgen init_surf jdata.
+    
+    Returns
+    -------
+    Argument
+        dpgen init_surf jdata arginfo
+    """
+    doc_init_surf = "Generate initial data for surface systems."
+    doc_stages = "Stages for `init_surf`."
+    doc_elements = "Atom types."
+    doc_potcars = "Path of POTCAR."
+    doc_cell_type = "Specifying which typical structure to be generated. **Options** include fcc, hcp, bcc, sc, diamond."
+    doc_super_cell = "Size of supercell."
+    doc_from_poscar = "Deciding whether to use a given poscar as the beginning of relaxation. If it's true, keys (`cell_type`, `latt`) will be aborted. Otherwise, these two keys are **necessary**."
+    doc_from_poscar_path = "Path of POSCAR for VASP or STRU for ABACUS. **Necessary** if `from_poscar` is true."
+    doc_latt = "Lattice constant for single cell."
+    doc_layer_numb = "Number of equivalent layers of slab."
+    doc_z_min = "Thickness of slab without vacuum (Angstrom). If the layer_numb and z_min are all set, the z_min value will be ignored."
```

Comment:
What is "Thickness of slab without vacuum" ? Do you mean the distance between the top layer and bottom layer atoms? 

"If the layer_numb and z_min are all set, the z_min value will be ignored."
Should be a typo